test/OpenTelemetry.DynamicActivityBinding/System.Diagnostics.DiagnosticSource.Tests/System.Diagnostics.DiagnosticSource.Tests.csproj

```
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <TargetFrameworks>$(NetCoreAppCurrent);net48-Windows_NT</TargetFrameworks>
```

This might solve part of the issues in the pipeline
```suggestion
    <TargetFrameworks>$(NetCoreAppCurrent);net48</TargetFrameworks>
```

---

And, since you are testing in linux and windows, you will probably have to make a condition in OS to add `net48`

---

These are not meant to build. I wanted to merge this to preserve the history (as described) and getting these to work correctly will be a subsequent process. They are currently not referenced or sued anywhere, so I am not sure how they got picked up. I'll rename the project file and try again. :)

---

They are picked up automatically by CI "dotnet build":
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/09bed2ab67178639023a9260b8620d22de6be802/.azure-pipelines/unit-tests.yml#L47-L55

Adding a line preceeded by `!` can be used to exclude specific projects, see https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/09bed2ab67178639023a9260b8620d22de6be802/.azure-pipelines/integration-tests.yml#L269-L273